components/molecule/accordion/src/index.js

```diff
@@ -57,6 +62,7 @@ MoleculeAccordion.propTypes = {
    * Array with tab indexes to be opened by default
    */
   defaultOpenedTabs: PropTypes.arrayOf(PropTypes.Number),
+  forceClose: PropTypes.bool,
```

Comment:
as long as i know, it'll be a problem because if yo start navigating and use the forceClose boolean prop (true), then yo continue navigating and if you want to force it again you need to change twice (false, and true) to re-forceClose all tabs.

The right strategy might be to be a controlled-uncontrolled component combining `defaultOpenedTabs` and`openedTabs` also. I'll take it.